### PR TITLE
feat: adding balance locked, reserved and free

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/ticket/issues/1201

Adding locked, reserved and free balances. 
Unable to add bonded, staked and vesting as pallets are not implemented. 

## How to test:

Run yarn test && yarn test:integration

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: packages/core/src/balance/Balance.chain.ts [hunk 1]
```diff
@@ -10,57 +10,59 @@
...
 import { BlockchainApiConnection } from '@kiltprotocol/chain-helpers'
+
 import Identity from '../identity/Identity'
 import BalanceUtils from './Balance.utils'
 
 /**
- * Fetches the current balance of the account with [accountAddress].
- * <B>Note that balance amount is in Femto-Kilt (1e-15)and must be translated to Kilt-Coin</B>.
+ * Fetches the current balances of the account with [accountAddress].
+ * <B>Note that the balance amounts is in Femto-Kilt (1e-15)and must be translated to Kilt-Coin</B>.
```
Comment (LeonFLK) [inline packages/core/src/balance/Balance.chain.ts:25]:
```suggestion
 * <B>Note that the balance amounts are in Femto-Kilt (1e-15)and must be translated to Kilt-Coin</B>.
```

File: packages/core/src/balance/Balance.chain.ts [hunk 2]
```diff
@@ -10,57 +10,59 @@
...
-    ).data.free.toString()
-  )
+
+  return (await blockchain.api.query.system.account(accountAddress)).data
 }
 
 /**
  * Attaches the given [listener] for balance changes on the account with [accountAddress].
- * <B>Note that balance amount is in Femto-Kilt (1e-15) and must be translated to Kilt-Coin</B>.
+ * <B>Note that the balance amounts is in Femto-Kilt (1e-15) and must be translated to Kilt-Coin</B>.
```
Comment (LeonFLK) [inline packages/core/src/balance/Balance.chain.ts:52]:
```suggestion
 * <B>Note that the balance amounts are in Femto-Kilt (1e-15) and must be translated to Kilt-Coin</B>.
```

**Review by LeonFLK — APPROVED** [review]
Do you think we should have tests that take into account the other `Balances` fields besides `free`?

Otherwise
LGTM

**Comment by Dudleyneedham** [comment]
At the moment, we don't have the pallets implemented in the node to actually test the different balances.

**Pull request merged**